### PR TITLE
Addresses #2012 Edit 'olfactory granule cell'

### PR DESCRIPTION
**Pull request by bvarner-ebi**
Addresses #2012 Edit 'olfactory granule cell'
See comment: https://github.com/obophenotype/cell-ontology/issues/2012#issuecomment-1663966444

### CODE REVIEW

File: src/ontology/cl-edit.owl [hunk 1]
```diff
@@ -8791,10 +8792,17 @@ SubClassOf(Annotation(oboInOwl:is_inferred "true") obo:CL_0000625 obo:CL_0000791
...
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "doi:10.1038/s41598-018-27692-8") oboInOwl:hasExactSynonym obo:CL_0000626 "olfactory bulb granule cell")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "doi:10.1038/s41598-018-27692-8") rdfs:comment obo:CL_0000626 "Granule cells are the most abundant neuronal population in the olfactory bulb and are continuously renewed throughout life.")
 AnnotationAssertion(rdfs:label obo:CL_0000626 "olfactory granule cell")
-EquivalentClasses(obo:CL_0000626 ObjectIntersectionOf(obo:CL_0000120 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0002264)))
-SubClassOf(obo:CL_0000626 obo:CL_0000120)
+EquivalentClasses(obo:CL_0000626 ObjectIntersectionOf(obo:CL_0000120 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0005378)))
+SubClassOf(obo:CL_0000626 obo:CL_0000099)
+SubClassOf(obo:CL_0000626 ObjectSomeValuesFrom(obo:RO_0002103 obo:CL_1001502))
+SubClassOf(obo:CL_0000626 ObjectSomeValuesFrom(obo:RO_0002103 obo:CL_1001503))
+SubClassOf(obo:CL_0000626 ObjectSomeValuesFrom(obo:RO_0002120 obo:CL_1001502))
```
Comment (aleixpuigb) [inline src/ontology/cl-edit.owl:8803]: This axiom is duplicated

File: src/ontology/cl-edit.owl [hunk 2]
```diff
@@ -8791,10 +8792,17 @@ SubClassOf(Annotation(oboInOwl:is_inferred "true") obo:CL_0000625 obo:CL_0000791
...
+SubClassOf(obo:CL_0000626 ObjectSomeValuesFrom(obo:RO_0002120 obo:CL_1001503))
```
Comment (aleixpuigb) [inline src/ontology/cl-edit.owl:8804]: This axiom is duplicated

**Review by aleixpuigb — CHANGES_REQUESTED** [review]
Please, see inline comments.

File: src/ontology/cl-edit.owl (hunk 1 above)
Comment (bvarner-ebi) [reply src/ontology/cl-edit.owl:8803]: Can you clarify which axiom you see duplicated?

**Review by aleixpuigb — APPROVED** [review]

**Pull request merged**